src/backend/marsha/core/views.py

```diff
@@ -30,6 +35,26 @@
 logger = getLogger(__name__)
 
 
+def exception_handler(exc, context):
```

Comment:
It's common to have an `exceptions.py` file in the app. Maybe this would fit better there.

Reply:
I can move it. In DRF this handler is in the `views` module.

Reply:
oh ok! leave it here then :man_shrugging: